file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [12-Apr-2025 03:17:31 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/wp-github-commits/wp-github-commits.php:258
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wp-github-commi...')
#2 /internal/eval.php(20): activate_plugin('wp-github-commi...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/wp-github-commits/wp-github-commits.php on line 258
[12-Apr-2025 03:17:39 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/wp-github-commits/wp-github-commits.php:258
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wp-github-commi...')
#2 /internal/eval.php(20): activate_plugin('wp-github-commi...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/wp-github-commits/wp-github-commits.php on line 258
[12-Apr-2025 03:17:46 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/wp-github-commits/wp-github-commits.php:258
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('wp-github-commi...')
#2 /internal/eval.php(20): activate_plugin('wp-github-commi...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/wp-github-commits/wp-github-commits.php on line 258
[12-Apr-2025 03:18:25 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT COUNT(id) as totalrows from wp_mynearbyplaces_category</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT COUNT ( `id` ) as `totalrows` from `wp_mynearbyplaces_category` | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_mynearbyplaces_category
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('SELECT COUNT(id...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('SELECT COUNT(id...')
#3 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('SELECT COUNT(id...')
#4 /wordpress/wp-content/plugins/wp-nearby-places-basic/nearbyplaces.php(57): wpdb->get_results('SELECT COUNT(id...')
#5 /wordpress/wp-includes/class-wp-hook.php(324): wnp_mynearbyplaces_activate(false)
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wp-nea...', false)
#9 /internal/eval.php(20): activate_plugin('wp-nearby-place...')
#10 {main}</pre>
 for query SELECT COUNT(id) as totalrows from wp_mynearbyplaces_category made by activate_plugin, do_action('activate_wp-nearby-places-basic/nearbyplaces.php'), WP_Hook->do_action, WP_Hook->apply_filters, wnp_mynearbyplaces_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[12-Apr-2025 03:18:25 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT COUNT(id) as totalrows from wp_mynearbyplaces_sub_category</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT COUNT ( `id` ) as `totalrows` from `wp_mynearbyplaces_sub_category` | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_mynearbyplaces_sub_category
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('SELECT COUNT(id...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('SELECT COUNT(id...')
#3 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('SELECT COUNT(id...')
#4 /wordpress/wp-content/plugins/wp-nearby-places-basic/nearbyplaces.php(89): wpdb->get_results('SELECT COUNT(id...')
#5 /wordpress/wp-includes/class-wp-hook.php(324): wnp_mynearbyplaces_activate(false)
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wp-nea...', false)
#9 /internal/eval.php(20): activate_plugin('wp-nearby-place...')
#10 {main}</pre>
 for query SELECT COUNT(id) as totalrows from wp_mynearbyplaces_sub_category made by activate_plugin, do_action('activate_wp-nearby-places-basic/nearbyplaces.php'), WP_Hook->do_action, WP_Hook->apply_filters, wnp_mynearbyplaces_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[12-Apr-2025 03:18:36 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT COUNT(id) as totalrows from wp_mynearbyplaces_category</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT COUNT(id) as totalrows from wp_mynearbyplaces_category | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_mynearbyplaces_category.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('SELECT COUNT(id...')
#2 /wordpress/wp-content/plugins/wp-nearby-places-basic/nearbyplaces.php(57): wpdb->get_results('SELECT COUNT(id...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): wnp_mynearbyplaces_activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wp-nea...', false)
#7 /internal/eval.php(20): activate_plugin('wp-nearby-place...')
#8 {main}</pre>
 for query SELECT COUNT(id) as totalrows from wp_mynearbyplaces_category made by activate_plugin, do_action('activate_wp-nearby-places-basic/nearbyplaces.php'), WP_Hook->do_action, WP_Hook->apply_filters, wnp_mynearbyplaces_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[12-Apr-2025 03:18:36 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT COUNT(id) as totalrows from wp_mynearbyplaces_sub_category</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT COUNT(id) as totalrows from wp_mynearbyplaces_sub_category | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_mynearbyplaces_sub_category.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('SELECT COUNT(id...')
#2 /wordpress/wp-content/plugins/wp-nearby-places-basic/nearbyplaces.php(89): wpdb->get_results('SELECT COUNT(id...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): wnp_mynearbyplaces_activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wp-nea...', false)
#7 /internal/eval.php(20): activate_plugin('wp-nearby-place...')
#8 {main}</pre>
 for query SELECT COUNT(id) as totalrows from wp_mynearbyplaces_sub_category made by activate_plugin, do_action('activate_wp-nearby-places-basic/nearbyplaces.php'), WP_Hook->do_action, WP_Hook->apply_filters, wnp_mynearbyplaces_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[12-Apr-2025 03:18:47 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT COUNT(id) as totalrows from wp_mynearbyplaces_category</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT COUNT(id) as totalrows from wp_mynearbyplaces_category | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_mynearbyplaces_category.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('SELECT COUNT(id...')
#2 /wordpress/wp-content/plugins/wp-nearby-places-basic/nearbyplaces.php(57): wpdb->get_results('SELECT COUNT(id...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): wnp_mynearbyplaces_activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wp-nea...', false)
#7 /internal/eval.php(20): activate_plugin('wp-nearby-place...')
#8 {main}</pre>
 for query SELECT COUNT(id) as totalrows from wp_mynearbyplaces_category made by activate_plugin, do_action('activate_wp-nearby-places-basic/nearbyplaces.php'), WP_Hook->do_action, WP_Hook->apply_filters, wnp_mynearbyplaces_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[12-Apr-2025 03:18:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT COUNT(id) as totalrows from wp_mynearbyplaces_sub_category</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT COUNT(id) as totalrows from wp_mynearbyplaces_sub_category | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_mynearbyplaces_sub_category.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3142): WP_SQLite_DB->query('SELECT COUNT(id...')
#2 /wordpress/wp-content/plugins/wp-nearby-places-basic/nearbyplaces.php(89): wpdb->get_results('SELECT COUNT(id...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): wnp_mynearbyplaces_activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wp-nea...', false)
#7 /internal/eval.php(20): activate_plugin('wp-nearby-place...')
#8 {main}</pre>
 for query SELECT COUNT(id) as totalrows from wp_mynearbyplaces_sub_category made by activate_plugin, do_action('activate_wp-nearby-places-basic/nearbyplaces.php'), WP_Hook->do_action, WP_Hook->apply_filters, wnp_mynearbyplaces_activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[12-Apr-2025 03:19:32 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>wp-tao</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6114
[12-Apr-2025 03:19:47 UTC] PHP Warning:  Undefined property: stdClass::$SIZE_MB in /wordpress/wp-content/plugins/wp-tao/includes/class-wptao-helpers.php on line 893
[12-Apr-2025 03:19:48 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>wp-tao</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6114
[12-Apr-2025 03:20:03 UTC] PHP Warning:  Undefined property: stdClass::$SIZE_MB in /wordpress/wp-content/plugins/wp-tao/includes/class-wptao-helpers.php on line 893
[12-Apr-2025 03:20:04 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>wp-tao</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6114
[12-Apr-2025 03:21:26 UTC] PHP Fatal error:  Uncaught Error: Undefined constant "home_url" in /wordpress/wp-content/plugins/warcraft-bundle/wb-main-fork.php:54
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('warcraft-bundle...')
#2 /internal/eval.php(20): activate_plugin('warcraft-bundle...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/warcraft-bundle/wb-main-fork.php on line 54
[12-Apr-2025 03:23:00 UTC] PHP Fatal error:  Uncaught Error: Undefined constant "home_url" in /wordpress/wp-content/plugins/warcraft-bundle/wb-main-fork.php:54
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('warcraft-bundle...')
#2 /internal/eval.php(20): activate_plugin('warcraft-bundle...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/warcraft-bundle/wb-main-fork.php on line 54
[12-Apr-2025 03:24:40 UTC] PHP Fatal error:  Uncaught Error: Undefined constant "home_url" in /wordpress/wp-content/plugins/warcraft-bundle/wb-main-fork.php:54
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('warcraft-bundle...')
#2 /internal/eval.php(20): activate_plugin('warcraft-bundle...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/warcraft-bundle/wb-main-fork.php on line 54
[12-Apr-2025 03:26:42 UTC] PHP Parse error:  syntax error, unexpected token "new" in /wordpress/wp-content/plugins/woocommerce-pinterest-button-extension/sod_pinterest.php on line 111
[12-Apr-2025 03:26:51 UTC] PHP Parse error:  syntax error, unexpected token "new" in /wordpress/wp-content/plugins/woocommerce-pinterest-button-extension/sod_pinterest.php on line 111
[12-Apr-2025 03:26:59 UTC] PHP Parse error:  syntax error, unexpected token "new" in /wordpress/wp-content/plugins/woocommerce-pinterest-button-extension/sod_pinterest.php on line 111

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #1 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"wow-dkp-system"},"options":{"activate":true}}) : Could not download "https://downloads.wordpress.org/plugin/wow-dkp-system.latest-stable.zip".
  				Check if the URL is correct and the server is reachable.
  				If it is reachable, the server might be blocking the request.
  				Check the browser console and network tabs for more information.
  
  				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
  
  				This means the server that hosts your file does not allow requests from other sites
  				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
  				cross-origin file downloads. Learn more about CORS at
  				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
  
  				If your file is on GitHub, load it from "raw.githubusercontent.com".
  				Here's how to do that:
  
  				1. Start with the original GitHub URL of the file. For example:
  				https://github.com/username/repository/blob/branch/filename.
  				2. Replace "github.com" with "raw.githubusercontent.com".
  				3. Remove the "/blob/" part of the URL.
  
  				The resulting URL should look like this:
  				https://raw.githubusercontent.com/username/repository/branch/filename
  
  				Error:
  				Error: Could not download "https://downloads.wordpress.org/plugin/wow-dkp-system.latest-stable.zip"
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: Error: Could not download "https://downloads.wordpress.org/plugin/wow-dkp-system.latest-stable.zip".
    				Check if the URL is correct and the server is reachable.
    				If it is reachable, the server might be blocking the request.
    				Check the browser console and network tabs for more information.
    
    				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
    
    				This means the server that hosts your file does not allow requests from other sites
    				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
    				cross-origin file downloads. Learn more about CORS at
    				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
    
    				If your file is on GitHub, load it from "raw.githubusercontent.com".
    				Here's how to do that:
    
    				1. Start with the original GitHub URL of the file. For example:
    				https://github.com/username/repository/blob/branch/filename.
    				2. Replace "github.com" with "raw.githubusercontent.com".
    				3. Remove the "/blob/" part of the URL.
    
    				The resulting URL should look like this:
    				https://raw.githubusercontent.com/username/repository/branch/filename
    
    				Error:
    				Error: Could not download "https://downloads.wordpress.org/plugin/wow-dkp-system.latest-stable.zip"
        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)
  }
}

Node.js v23.11.0
